api/database/boltdb/export.go

```diff
@@ -8,9 +8,30 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+func backupMetadata(connection *bolt.DB) (map[string]interface{}, error) {
+	buckets := map[string]interface{}{}
+
+	err := connection.View(func(tx *bolt.Tx) error {
+		err := tx.ForEach(func(name []byte, bucket *bolt.Bucket) error {
+			bucketName := string(name)
+			bucket = tx.Bucket([]byte(bucketName))
+			seqId := bucket.Sequence()
+			buckets[bucketName] = int(seqId)
+			return nil
+		})
+
+		return err
+	})
+
+	return buckets, err
+}
```

Comment:
I  thought we added this because the interface was originally defined in CE and you wanted to add the ability to not backup metadata?   Now that we are in CE can we now change the original interface and use the interface method?

Reply:
I copied it as is from EE. It is is not this method but ExportRaw method which we were talking about. IMO, we should have imported a few files from here in the EE code.